user-service/src/main/java/com/local_dating/user_service/presentation/controller/UserController.java

```diff
@@ -133,11 +133,12 @@ public ResponseEntity saveProfile(final Authentication authentication, @RequestB
     }
 
     @PatchMapping(value = "/profile")
-    public ResponseEntity updateProfile(final Authentication authentication, @RequestBody final List<UserProfileDTO> userProfileDTOList) throws Exception {
+    public void updateProfile(final Authentication authentication, @RequestBody final List<UserProfileDTO> userProfileDTOList) throws Exception {
+    //public ResponseEntity updateProfile(final Authentication authentication, @RequestBody final List<UserProfileDTO> userProfileDTOList) throws Exception {
         String userId = (String) authentication.getPrincipal();
         userProfileService.updateProfile(userId, userProfileDTOList);
 
```

Comment:
Optional.of((String) authentication.getPrincipal()).map(s -> userProfileService.updateProfile(s, userProfileDTOList))